Networking/Networking/Remote/ProductReviewsRemote.swift

```diff
@@ -0,0 +1,92 @@
+import Foundation
+
+/// Product reviews: Remote Endpoints
+///
+public final class ProductReviewsRemote: Remote {
+
+    // MARK: - Product Reviews
+
+    /// Retrieves all of the `ProductReview` available. We rely on the endpoint
+    /// defaults for `sorting` and `orderby`: `date_gmt` and `asc`
+    ///
+    /// - Parameters:
+    ///     - siteID: Site for which we'll fetch remote products.
+    ///     - context: view or edit. Scope under which the request is made;
+    ///                determines fields present in response. Default is view.
+    ///     - pageNumber: Number of page that should be retrieved.
+    ///     - pageSize: Number of Orders to be retrieved per page.
+    ///     - completion: Closure to be executed upon completion.
+    ///
+    public func loadAllProductReviews(for siteID: Int,
+                                context: String? = nil,
```

Comment:
does it make sense to make an enum for `context` param? otherwise, the caller could potentially pass any value (not sure if there are any consequences)